docs/source/user/troubleshoot.rst

```diff
@@ -604,20 +613,29 @@ presented for brevity):
     $2 = 124
     (gdb) p c
     $3 = 0
-    (gdb) n
-    8           d = (a, b, c)
-    (gdb) p c
-    $4 = 287.81999999999999
+    (gdb) b 9
+    Breakpoint 2 at 0x7fb73d1f7287: file demo_gdb.py, line 9.
+    (gdb) c
+    Continuing.
+
+    Breakpoint 2, __main__::foo_241[abi:c8tJTC_2fWgEeGLSgydRTQUgiqKEZ6gEoDvQJmaQIA](long long) (a=123) at demo_gdb.py:9
+    9           print(a, b, c, d)
+    (gdb) info locals
+    b = 124
+    c = 287.81999999999999
+    d = {f0 = 123, f1 = 124, f2 = 287.81999999999999}
+
 
 It can be seen in the above example that execution of the code is paused at the
 location of the ``gdb()`` function call at end of the ``numba_gdb_breakpoint``
 function (this is the Numba internal symbol registered as breakpoint with
-``gdb``). Issuing a ``step`` at this point moves to the stack frame of the
+``gdb``). Issuing a ``step`` twice at this point moves to the stack frame of the
 compiled Python source. From there, it can be seen that the variables ``a`` and
 ``b`` have been evaluated but ``c`` has not, as demonstrated by printing their
 values, this is precisely as expected given the location of the ``gdb()`` call.
-Issuing a ``next`` then evaluates line ``7`` and ``c`` is assigned a value as
-demonstrated by the final print.
+Issuing a ``break`` on line 9 and then continuing evaluates line ``7`` and ``c``
+is assigned a value as demonstrated by ``info locals`` when the breakpoint it
```

Comment:
I have a hard time parsing the "continuing evaluates ... is assigned a value"

Reply:
I've reworded this in cf6dee6

Reply:
And again, as discussed OOB in 6a9c8a1 .